core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/util/StringToolkit.java

```diff
@@ -36,16 +36,16 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
+import java.nio.charset.Charset;
```

Comment:
Got to update the copyright year to 2023 here as well, i.e.: 
 Copyright (c) 2018, 2023, Oracle and/or its affiliates. All rights reserved.